pages/learn/glossary.md

```diff
@@ -55,6 +55,10 @@ The essence of JSON Hyper-Schema lies in its ability to define links and actions
 
 In other words, JSON Hyper-Schema extends JSON Schema by introducing features for creating hypermedia controls. This facilitates the creation of interactive APIs and ensures compatibility with existing JSON HTTP APIs, maintaining a seamless integration. It adds a layer of interactivity to JSON documents, making it easier to interact with remote JSON resources.
 
+### JSON pointer
+
+JSON Pointer is a format for data exchange between systems, defining a string for accessing and modifying values within a JSON document. By using JSON Pointer, we can precisely reference and retrieve specific values within a JSON document. The [subschema](#subschema) is typically identified via a JSON Pointer, or equivalent syntax, allowing for targeted validation and manipulation of data within the JSON document. 
```

Comment:
It'd be good to link to the JSON Pointer spec (RFC 6901).

Also, JSON Pointer merely identifies a location within a document. It doesn't (itself) access or modify values, though it _is_ used by other standards to accomplish those tasks.

Reply:
Thanks for adding the link, but the rest has not been addressed.  JSON Pointer does not modify documents.  It only serves to indicate a location within a document.